docs/marginaleffects.ipynb

```diff
@@ -0,0 +1,371 @@
+{
```

Comment:
<p>For the non-expert readers, I suggest you explain how you reach this conclusion from the table :) For example, is it the -0.81?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/py-econometrics/pyfixest/pull/473/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>